Orange/widgets/data/owconcatenate.py

```diff
@@ -115,15 +115,9 @@ def __init__(self):
         cb.disables.append(ibox)
         cb.makeConsistent()
 
-        gui.separator(self.controlArea, 8)
-
-        box = gui.widgetBox(self.controlArea, True, orientation="horizontal",)
-        box.layout().addWidget(self.report_button)
-        gui.separator(box, 20)
-        gui.button(
-            box, self, self.tr("Apply Changes"),
-            callback=self.apply, default=True
-        )
+        gui.separator(self.buttonsArea, 20)
+        gui.button(self.buttonsArea, self, "Apply Changes",
```

Comment:
I just meant, since you touched these lines, why not make it an `auto_commit` button instead of a simple one?